tests/dspa_v2_test.go

```diff
@@ -29,7 +29,16 @@ import (
 
 var _ = Describe("A successfully deployed DSPA", func() {
 
-	podCount := 8
+	podCount := 6
+
+	var isTestDspa bool
+
+	BeforeEach(func() {
```

Comment:
Could you rebase the PR? With the refactoring changes merged in, this function might not be supported in testify.